Formula/solarus.rb

```diff
@@ -27,19 +27,22 @@ class Solarus < Formula
   depends_on "sdl2_image"
   depends_on "sdl2_ttf"
 
+  on_linux do
+    depends_on "mesa"
```

Comment:
Odd that there is no linkage to libGL.so. May need to check to make sure it is properly picking it up.